charts/kyverno/templates/_helpers.tpl

```diff
@@ -116,3 +116,30 @@ maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
 {{- define "kyverno.imagePullSecret" }}
 {{- printf "{\"auths\":{\"%s\":{\"auth\":\"%s\"}}}" .registry (printf "%s:%s" .username .password | b64enc) | b64enc }}
 {{- end }}
+
+
+{{- define "kyverno.resourceFilters" -}}
+{{- $resourceFilters := .Values.config.resourceFilters }}
+{{- if .Values.excludeKyvernoNamespace }}
+  {{- $resourceFilters = prepend .Values.config.resourceFilters (printf "[*,%s,*]" (include "kyverno.namespace" .)) }}
+{{- end }}
+{{- tpl (join "" $resourceFilters) . }}
+{{- end }}
+
+{{- define "kyverno.webhooks" -}}
+{{- $excludeDefault := dict "key" "kubernetes.io/metadata.name" "operator" "NotIn" "values" (list (include "kyverno.namespace" .)) }}
+{{- $newWebhook := list }}
+{{- range $webhook := .Values.config.webhooks }}
+  {{- $namespaceSelector := default dict $webhook.namespaceSelector }}
+  {{- $matchExpressions := default list $namespaceSelector.matchExpressions }}
+  {{- if $.Values.excludeKyvernoNamespace }}
```

Comment:
If this is `false`, we shouldn't need to iterate over the slice at all.
I find it strange to unpack/repack in case this is ` false`.

Reply:
So I put the logic into existing if/else if in main configmap so this helper function only called if both webhooks and excluding kyverno namespace.  Otherwise if only webhooks, just use those (no function) or just enable default then use that.